python_scripts/cross_validation_learning_curve.py

```diff
@@ -104,7 +104,7 @@
 #
 # If we achieve a plateau and adding new samples in the training set does not
 # reduce the testing error, we might have reached the Bayes error rate using the
-# available model. Using a more complex model might be the only possibility to
+# available model. Using a more expressive model might be the only possibility to
 # reduce the testing error further.
```

Comment:
@ogrisel If a model reached Bayes error then isn’t that error is irreducible meaning if we even select a more expressive model that won’t improve our test error?

Reply:
Wording fixed in [bcc45f3](https://github.com/INRIA/scikit-learn-mooc/pull/780/commits/bcc45f3adb6df161780ab042d9b222e2d7259066).